### DISCUSSION TITLE
Scripts not running in Playground

### DISCUSSION THREAD

**Discussion by eddyloewen** [tailwindlabs/tailwindcss#13676] [Help]
I wanted to use some script blocks inside the Tailwind Playground. I can see the sandbox="allow-scripts" attribute on the iframe but for some reason the scripts won't run.
Is that intended behaviour? Or should that be possible?
https://play.tailwindcss.com/9V1tqWBPyn

**Comment by wongjn** [comment]
Custom JavaScript is not expected to be able to run in the Playground.

**Reply by eddyloewen** [reply]
How do you know?

**Reply by wongjn** [reply]
From past experience in using it – I've never got any `<script>` elements to execute.

**Reply by eddyloewen** [reply]
But it could still be "just" a bug, no?

**Reply by wongjn** [reply]
I guess so! Sorry, I should rephrase my initial statement: "*I* don't expect `<script>` elements to execute in the Playground."
